src/columnsTypes/types/Label/Component.jsx

```diff
@@ -78,6 +97,8 @@ export const LabelsChooser = ({view, pos, node}) => {
   );
 
   const handleClose = (currentChosenLabels) => {
+    if (onClose) onClose();
```

Comment:
is the below a default on close procedure?

Reply:
yep, added a comment.
the labels dropdown can "live" in two places - labels cell and filters, each one has different close procedure